src/router-view.ts

```
@@ -0,0 +1,42 @@
+/// <reference types="./../lib/index.d.ts" />
+
+import { global } from './global'
+
+export class RouterView extends HTMLElement {
+  constructor() {
+    super()
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)